### PR TITLE
v2v: add TC 'convert guest with sata disk'

### PR DESCRIPTION
**Pull request by vwu-vera**
Please check bz#1677608.

### CODE REVIEW

**Comment by vwu-vera** [comment]
(1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.sata.esx_65.rhev.rhv_upload.it_vddk.vpx_uri: PASS (1352.13 s)

**Comment by vwu-vera** [comment]
@xiaodwan Please help to review. Thanks.

File: v2v/tests/cfg/function_test_esx.cfg [hunk 1]
```diff
@@ -676,6 +676,11 @@
         - gt_16_disks:
             only esx_70
             main_vm = 'VM_NAME_ESX_GT_16_DISKS_V2V_EXAMPLE'
+        - sata:
+            only esx_65
+            only rhev
```
Comment (xiaodwan) [inline v2v/tests/cfg/function_test_esx.cfg:681]: This line is not neccessary.

Comment (xiaodwan) [inline v2v/tests/cfg/function_test_esx.cfg:681]: This line is not necessary.

File: v2v/tests/cfg/function_test_esx.cfg [hunk 2]
```diff
@@ -676,6 +676,11 @@
...
+            version_required = "[libvirt-7.8.0-1.module+el8.6.0+12978+7d7a0321,)"
```
Comment (xiaodwan) [inline v2v/tests/cfg/function_test_esx.cfg:682]: It should be "[libvirt-7.8.0-1,)". 
It's not a simple copy of the fixed version.  We should think of the whole RHEL versions and make sure the comparison
is meaningful.

Comment (vwu-vera) [reply v2v/tests/cfg/function_test_esx.cfg]: Thanks. Please review the modification.

**Review by xiaodwan — APPROVED** [review]

**Pull request merged**